### PR TITLE
 Move command over to new v2 structure

### LINKED ISSUE #1: Move command over to new v2 structure
The following changes need to be made to move the command over to the v2 structure:

- [ ] Make sure the correct framework is required:
  ```
  composer require wp-cli/wp-cli:^2
  ```
- [ ] Require the testing framework as a dev dependency:
  ```
  composer require --dev wp-cli/wp-cli-tests:^2
  ```
- [ ] Use the `.travis.yml` file from `wp-cli/wp-cli`:
  ```
  wget https://raw.githubusercontent.com/wp-cli/wp-cli/master/.travis.yml
  ```
- [ ] Add the default script configuration to Composer file:
  ```
    "scripts": {
        "lint": "run-linter-tests",
        "phpcs": "run-phpcs-tests",
        "phpunit": "run-php-unit-tests",
        "behat": "run-behat-tests",
        "prepare-tests": "install-package-tests",
        "test": [
            "@lint",
            "@phpcs",
            "@phpunit",
            "@behat"
        ]
    },
  ```
- [ ] Remove scaffolded binary files:
  ```
  git rm bin/install-package-tests.sh
  git rm bin/test.sh
  ```
- [ ] Remove scaffolded Behat setup:
  ```
  git rm features/bootstrap/*
  git rm features/extra/*
  git rm features/steps/*
  ```
- [ ] Remove scaffolded Behat tags util script:
  ```
  git rm utils/behat-tags.php
  ```
- [ ] Add command packages that are needed for Behat tests as `--dev` dependencies.
  The following commands are already available, anything else needs to be explicitly required:
  - `cli *`
  - `config *`
  - `core *`
  - `eval`
  - `eval-file`
  - `help`
- [ ] Update all dependencies:
  ```
  composer update
  ```
- [ ] *Optional* - Add PHPCS rule set to enable CS & compatibility sniffing:
  ```
  wget https://raw.githubusercontent.com/wp-cli/wp-cli/master/phpcs.xml.dist
  ```
- [ ] Run and adapt tests to make sure they all pass:
  ```
  composer test
  ```

### PR DESCRIPTION
**Pull request by wojsmol**
fixes https://github.com/wp-cli/google-sitemap-generator-cli/issues/1

### CODE REVIEW

**Comment by wojsmol** [comment]
Test almost pass - https://travis-ci.com/wojsmol/google-sitemap-generator-cli/builds/81593959 - previous issue discussed on slack was caused by a typo.

**Comment by wojsmol** [comment]
Any hint on converting to PSR-4 will by appreciated.

**Comment by wojsmol** [comment]
PR is ready for review.

File: composer.json [hunk 1]
```diff
@@ -1,15 +1,44 @@
...
+            "name": "WP-CLI Team",
+            "homepage": "http://github.com/wp-cli",
+            "role": "Developer"
+        }
+    ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.4",
+        "wp-cli/wp-cli": "^2",
+        "wp-cli/wp-cli-tests": "^2.0.7"
```
Comment (schlessera) [inline composer.json]: This should move into `require-dev` below.

**Review by schlessera — CHANGES_REQUESTED** [review]
@wojsmol Only 1 minor change, and we're good to go!

File: composer.json (hunk 1 above)
Comment (wojsmol) [reply composer.json]: fixed in https://github.com/wp-cli/google-sitemap-generator-cli/pull/2/commits/8d77acba389ffc674014340122863a1168bb7b1a

**Review by schlessera — APPROVED** [review]

**Pull request merged**